Sources/StreamChatUI/ChatChannelList/ChatChannelListItemView.swift

```diff
@@ -188,6 +217,17 @@ open class ChatChannelListItemView: _View, ThemeProvider, SwiftUIRepresentable {
 
         unreadCountView.content = content?.channel.unreadCount ?? .noUnread
         unreadCountView.invalidateIntrinsicContentSize()
+        
+        let checkmarkContent = previewMessageDeliveryStatus
+        previewMessageDeliveryStatusView.content = checkmarkContent
+        previewMessageDeliveryStatusView.isHidden = checkmarkContent == nil
+        
+        if let status = checkmarkContent?.deliveryStatus {
+            bottomContainer.insertArrangedSubview(
+                previewMessageDeliveryStatusView,
+                at: status == .pending || status == .failed ? 0 : 1
+            )
+        }
```

Comment:
Then, this makes it more readable as well:
```suggestion
        previewMessageDeliveryStatusView.content = previewMessageDeliveryStatus 
        previewMessageDeliveryStatusView.isHidden = previewMessageDeliveryStatus == nil
        
        if let status = previewMessageDeliveryStatus {
            bottomContainer.insertArrangedSubview(
                previewMessageDeliveryStatusView,
                at: status == .pending || status == .failed ? 0 : 1
            )
        }
```

Reply:
This way `previewMessageDeliveryStatus` is calculated 3 times

Reply:
Please, read my comment above 🙂